taxonomies/categories.txt

```diff
@@ -80850,6 +80894,9 @@ ciqual_food_name:fr:Porc, longe, crue
 en:Frozen pork loin
 fr:Longe de porc surgelée
 
+<en:Pork loin
+es:Lomo
+
```

Comment:
Isn't it that lomo in Spanish = loin in English?

What about the following?
 - put "es:Lomo de cerdo" as translation of "en:Pork loin". https://www.wikidata.org/wiki/Q7230136
 - eventually, create a parent "en:loin" and add "es:lomo" as its Spanish translation. And then, add "<en:loin" above "en:Pork loin". https://www.wikidata.org/wiki/Q115733508

Reply:
I added es:Lomo de cerdo under Pork Loin.
However, the 'es:Lomo" refers to a Spanish delicacy that is 'Cured pork loin'. Also called 'Lomo embuchado'. I added those two tags to the entry, starting with the 'en:' tag --> 'en:Cured pork loin'